celer/homotopy.py

```diff
@@ -302,7 +302,10 @@ def celer_path(X, y, pb, eps=1e-3, n_alphas=100, alphas=None, l1_ratio=1.0,
                     X_data, X_indices, X_indptr, X_sparse_scaling,
                     weights, len(grp_ptr) - 1, np.zeros(1, dtype=np.int32),
                     X_sparse_scaling.any())
-            theta *= alpha * l1_ratio / scal
+            if scal / n_samples >= alpha * l1_ratio:
+                theta *= alpha * l1_ratio / scal
+            else:
+                theta /= n_samples
```

Comment:
This is because we do not ``create_dual_point``  in the first iteration of Lasso.

I am not sure about it as I assume that the problem is Lasso therefore we treat  (for example) ``Logreg`` with ``alpha_max < alpha`` the same way.